src/components/blocks/documentsBlock/DocumentsBlock.tsx

```diff
@@ -51,7 +51,7 @@ export const DocumentsBlock = ({ countries, family }: IProps) => {
       id: doc.slug,
       cells: {
         year,
-        geography: family.geographies.map((geo) => getCountryName(geo, countries)).join(", "),
+        geography: family.geographies.map((geo) => getCountryName(geo, countries) || geo).join(", "),
```

Comment:
Note for later: we should be able to plug in the get subdivision name, maybe `?? getSubdivisionName(geo)`